openid-4-verifiable-presentations-1_0.md

```diff
@@ -2249,6 +2274,46 @@ The VP Token contains the base64url-encoded `DeviceResponse` CBOR structure as d
 
 See ISO/IEC TS 18013-7 Annex B [@ISO.18013-7] and ISO/IEC 23220-4 Annex C [@ISO.23220-4] for the latest examples on how to use the `presentation_submission` parameter and how to generate the Authorizaton Response for presenting Credentials in the mdoc format.
 
+### `Handover` and `SessionTranscript` Definitions
+
+#### Invocation via the Digital Credentials API
+
+If the presentation request is invoked using the Digital Credentials API, the `SessionTranscript` CBOR structure as defined in Section 9.1.5.1 in [ISO.18013-5] MUST be used with the following changes:
+
+* `DeviceEngagementBytes` MUST be `null`.
+* `EReaderKeyBytes` MUST be `null`.
+* `Handover` MUST be the `OID4VPDCAPIHandover` CBOR structure as defined below.
```

Comment:
As we had the naming discussion recently, do we want these values to be OPENID4VP... ?

Reply:
Not sure why anyone would confuse CBOR structure names with OIDs? But I can update the PR to use `OpenID4VPDigitalCredentialsAPIHandover` if this works for you?

Reply:
`OpenID4VPDCAPIHandover` is probably sufficient.

> Not sure why anyone would confuse CBOR structure names with OIDs?
I am 100% with you Oliver, but this has came up enough times, unfortunately.......